warp/src/main/java/com/schibsted/nmp/warp/theme/WarpDimensions.kt

```diff
@@ -41,6 +41,7 @@ object WarpDimensions {
 object WarpComponentDimensions {
     val stepIndicator = StepIndicatorDimensions
     val pillIcon = 16.dp
+    val smallButtonHeight = 35.dp
```

Comment:
shouldn't these dimensions be tied to the warp dimensions rather?

Reply:
These are custom